containers/changelog.md

```diff
@@ -45,6 +45,10 @@
 
 * Add `Data.Graph.flattenSCC1`. (Andreas Abel)
 
+* `NFData1`, `NFData2` instances for `Data.Graph`, `Data.IntMap`,
+  `Data.IntSet`, `Data.Map`, `Data.Sequence`, `Data.Set`, `Data.Tree` and
```

Comment:
`Data.Graph` could be misunderstood, `SCC` got the instance (not `Graph`). And `IntSet` is not affected.  
We can also just drop the `Data.` and say "...for `IntMap`, `Map`, `Set`, `Tree`, `SCC`", I think that's good enough.